docs/tutorials/tutorial-kafka.md

```diff
@@ -24,260 +24,267 @@ sidebar_label: "Load from Apache Kafka"
+10. In **Input tuning**, set **Use earliest offset** to `True`&mdash;this is very  important because you want to consume the data from the start of the stream. There are no other changes to make here, so click **Next: Publish**.
 
-This is the spec you have constructed.
-Feel free to go back and make changes in previous steps to see how changes will update the spec.
-Similarly, you can also edit the spec directly and see it reflected in the previous steps.
+    ![Data loader publish](../assets/tutorial-kafka-data-loader-08.png "Data loader publish")
 
-Once you are satisfied with the spec, click `Submit` and an ingestion task will be created.
+11. Name the datasource `kttm-kafka` and click **Next: Edit spec** to review your spec.
 
-![Tasks view](../assets/tutorial-kafka-data-loader-10.png "Tasks view")
+    ![Data loader spec](../assets/tutorial-kafka-data-loader-09.png "Data loader spec")
 
-You will be taken to the task view with the focus on the newly created supervisor.
+    The console presents the spec you've constructed. You can click the buttons above the spec to make changes in previous steps and see how the changes update the spec. You can also edit the spec directly and see it reflected in the previous steps.
+   
+12. Click **Submit** to create an ingestion task.
 
-The task view is set to auto refresh, wait until your supervisor launches a task.
+    Druid displays the task view with the focus on the newly created supervisor.
 
-When a tasks starts running, it will also start serving the data that it is ingesting.
+    The task view auto-refreshes, so wait until the supervisor launches a task. The status changes from **Pending** to **Running** as Druid starts to ingest data.
 
-Navigate to the `Datasources` view from the header.
+    ![Tasks view](../assets/tutorial-kafka-data-loader-10.png "Tasks view")
 
-![Datasource view](../assets/tutorial-kafka-data-loader-11.png "Datasource view")
+13. Navigate to the **Datasources** view from the header.
 
-When the `wikipedia-kafka` datasource appears here it can be queried. 
+    ![Datasource view](../assets/tutorial-kafka-data-loader-11.png "Datasource view")
 
-*Note:* if the datasource does not appear after a minute you might have not set the supervisor to read from the start of the stream (in the `Tune` step).
+    When the `kttm-kafka` datasource appears here, you can query it. See [Query your data](#query-your-data) for details.
 
-At this point, you can go to the `Query` view to run SQL queries against the datasource.
+    > If the datasource doesn't appear after a minute you might not have set the supervisor to read data from the start of the stream&mdash;the `Use earliest offset` setting in the **Tune** step. Go to the **Ingestion** page and terminate the supervisor using the **Actions** menu. [Load the sample data](#load-data-with-the-console-data-loader) again and apply the correct setting when you get to the **Tune** step.
```

Comment:
```suggestion
    > If the datasource doesn't appear after a minute you might not have set the supervisor to read data from the start of the stream&mdash;the `Use earliest offset` setting in the **Tune** step. Go to the **Ingestion** page and terminate the supervisor using the **Actions (...)** menu. [Load the sample data](#load-data-with-the-console-data-loader) again and apply the correct setting when you get to the **Tune** step.
```

Reply:
I don't like including the ellipsis and brackets in the instruction. I think it's clear what it's referring to in the UI. I'll add it to our style guide discussion list.